### ISSUE TITLE
Column drag-drop not working in RTL

### ISSUE THREAD

**Issue by arkogupta** [OfficeDev/office-ui-fabric-react#7230]
### Environment Information 
 
 
- __Package version(s)__: 
- __Browser and OS versions__: All

### Please provide a reproduction of the bug in a codepen:
 

Repro steps:

1. On the Fabric demo app page, click on the gear icon in the top right.
2. Select "Render in RTL"
3. Try dragging and dropping a column header.
4. The drop hint always comes at the first index, no matter where we drag a column header.
5. Reorder happens according to the drop hint index.

 
#### Actual behavior:
 
Drop hint always comes at the first index, no matter where we drag a column header.

![image](https://user-images.githubusercontent.com/15196876/49069105-74751080-f24e-11e8-8da2-f6c4bb22c3cd.png)

 
#### Expected behavior:

Drop hint should come according to the position where the column header is being dragged. The behavior should be the same as it is for LTR.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes
 
Requested priority: (Blocking, High, Normal, Low) Normal
 
Products/sites affected: (if applicable) SharePoint

**Comment by aftab-hassan** [comment]
@arkogupta Thank you for submitting this, and the clean repro steps. Seems like a valid bug withe the RTL.
Noticed you mentioned about submitting a fix for this. Is this something you would be interested in?

**Comment by arkogupta** [comment]
Yes @aftab-hassan. I am working on it and will raise the fix soon.
